src/sentry/tasks/sentry_apps.py

```diff
@@ -304,10 +312,12 @@ def send_and_save_webhook_request(url, sentry_app, app_platform_event):
             url=url, data=app_platform_event.body, headers=app_platform_event.headers, timeout=5
         )
 
-    except Timeout:
-        track_response_code("timeout", slug, event)
+    except (Timeout, ConnectionError) as e:
+        track_response_code(e.__class__.__name__, slug, event)
```

Comment:
@ceorourke Is the name going to be `Timeout` or `timeout`? We should keep the same casing as before so it doesn't put it into a new bucket

Reply:
it was `Timeout` so changed to lowercase them